test/formatters/formatters.outputBlockFormatter.js

```diff
@@ -1,5 +1,6 @@
-var assert = require('assert');
-var formatters = require('../lib/web3/formatters.js');
+var assert   = require('assert');
+//var assert = chai.assert;
```

Comment:
I did move in *many* places already from assert to chai assert.
I left it in 2 places because some tests failed due to deepEqual and I did not have time to investigate why.

By the way, I would recommend even to go for chai.expect instead of chai.assert
I did it on the new tests. This is once again something we just need to agree on.
My religion allows me using both :smile: